### PR TITLE
add admin API email endpoints

### PR DESCRIPTION
**Pull request by techknowlogick**
add email endpoint to admin API to ensure API parity with admin dashborad.

Note: swagger documentation hasn't yet been added.

### CODE REVIEW

File: routers/api/v1/admin/email.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	}
+
+	ctx.SetLinkHeader(int(maxResults), listOptions.PageSize)
+	ctx.SetTotalCountHeader(maxResults)
+	ctx.JSON(http.StatusOK, &results)
+}
+
+// GetEmail
+func GetEmail(ctx *context.APIContext) {
+	GetAllEmails(ctx)
```
Comment (lunny) [inline routers/api/v1/admin/email.go]: Is `Params(":email")` an accurate condition or inaccurate one? Maybe they should be treated differently.

Comment (techknowlogick) [reply routers/api/v1/admin/email.go]: It's accurate, as from routes the `{email}` route param is available to this route:
```
m.Group("/emails", func() {
...
  m.Group("/{email}", func() {
	  m.Get("", admin.GetEmail)
  })
})
```

Comment (lunny) [reply routers/api/v1/admin/email.go]: But if the router is `.../my123@123.com`, and there are two emails in the databases, one is `my123@123.com` which is correctly matched, another is `mmy123@123.com` which should not be matched but will.

Comment (techknowlogick) [reply routers/api/v1/admin/email.go]: ah, ok. I understand now. Thank you for details. Let me adjust this.

Comment (techknowlogick) [reply routers/api/v1/admin/email.go]: I've removed this endpoint, as it has too much overlap with Search Endpoint.

**Comment by techknowlogick** [comment]
attaching for posterity, this is the patch that adds for fetching specific emails (instead of using search endpoint) 
[individualEmail.patch](https://github.com/go-gitea/gitea/files/10963761/individualEmail.patch)

**Review by lunny — APPROVED** [review]

**Review by wolfogre — APPROVED** [review]

**Pull request merged**